### ISSUE TITLE
Feature Suggestion: Explosives

### ISSUE THREAD

**Issue by Badcholo** [simeonradivoev/MatterOverdrive#240]
I think a mod like this could use some very nice explosives like a Redmatter Explosive that creates a max size anomaly for like 3min and is much more destructive. An Antimatter Explosive to destroy a huge area, removing any Anomalys with it. I got these ideas from a voltz server i played on. It fits and ICBM is a dead mod, didn't make it to 1.7.10

**Comment by laserlemons** [comment]
This would be pretty cool, it seems like mods lately have been ignoring WMDs for some reason, which sucks because everyone loves a good explosion.

**Comment by Badcholo** [comment]
Yes and most of the explosive mods that are at 1.7.10 don't make any sense and look like garbage. 
I'm going to create a new modpack with MO as the main mod and everything will just revolve around it. I have even created a pixelmon modpack with MO, it's just epic. 
http://www.technicpack.net/modpack/espeon.766144

**Comment by Badcholo** [comment]
Bump to remind players that good explosions are needed.

**Comment by ajthemacboy** [comment]
Great idea, I wish there were more good explosives for large areas. Mekanism's obsidian TNT is pretty nice but it's not *that* much larger than vanilla TNT.
